Resources/Prototypes/Loadouts/Jobs/cargo.yml

```diff
@@ -1,3 +1,42 @@
+# Cargo technician
+- type: loadout
+  id: LoadoutCargoOuterWinterCargo
+  category: Jobs
+  cost: 2
+  exclusive: true
+  requirements:
+    - !type:CharacterJobRequirement
+      jobs:
+        - CargoTechnician
+  items:
+    - ClothingOuterWinterCargo
+
+- type: loadout
+  id: LoadoutCargoShoesBootsWinterCargo
+  category: Jobs
+  cost: 2
+  exclusive: true
+  requirements:
+    - !type:CharacterJobRequirement
+      jobs:
+        - CargoTechnician
```

Comment:
missing the Harpy blacklist for shoes! 
:trollface:

Reply:
i always forget these [ca87420](https://github.com/Simple-Station/Einstein-Engines/pull/616/commits/ca874202573ce4807e2b0947a6c306c85a5f0c95)